pkg/db/db.go

```diff
@@ -262,6 +293,7 @@ func (d *db) insert(ctx context.Context, rec record) (id int64, _ error) {
 		_ = tx.Rollback()
```

Comment:
nit: might be nice to track when the transaction succeeds or rolls back since we're tracking `beginTx`, but then again, that may make the events too noisy